README.md

```diff
@@ -126,6 +126,27 @@ npm run docker:run
 # Hooray! Leon is running
 ```
 
+### Docker guide for development
```

Comment:
There is already a guide for development with Docker inside `.github/CONTRIBUTING.md`.